x/config/configurator.go

```diff
@@ -97,61 +97,63 @@ func (c *Configurator) MustRegisterTransport(t TransportSpec) {
 	}
 }
 
-// RegisterChooser registers a ChooserSpec with the given Configurator. Returns
-// an error if the ChooserSpec is invalid.
+// RegisterPeerList registers a PeerListSpec with the given Configurator. Returns
+// an error if the PeerListSpec is invalid.
 //
 // If a chooser with the same name already exists, it will be replaced.
 //
-// Use MustRegisterChooser to panic in the case of registration failure.
-func (c *Configurator) RegisterChooser(s ChooserSpec) error {
+// Use MustRegisterPeerList to panic in the case of registration failure.
+func (c *Configurator) RegisterPeerList(s PeerListSpec) error {
```

Comment:
We've never mentioned the Configurator in the change log, it’s under `x/`, and we’ve never rolled it into production, so I’m good on just documenting the final version.